ctru-rs/src/services/cam.rs

```diff
@@ -307,7 +280,7 @@ pub trait Camera {
 
     /// Returns the maximum amount of transfer bytes based on the view size, trimming, and other
     /// modifications set to the camera
-    fn get_transfer_bytes(&self) -> crate::Result<u32> {
+    fn transfer_bytes(&self) -> crate::Result<u32> {
```

Comment:
minor nit, this sounds like it could be a verb ("transfer the bytes") but I guess the signature makes it more obvious. Maybe `transfer_byte_count()` is clearer?

Reply:
Oh, true. I hadn't seen it that way, thanks for the suggestion 👍